Sources/MapboxCoreNavigation/Result+Expected.swift

```diff
@@ -2,7 +2,8 @@ import Foundation
 @_implementationOnly import MapboxCommon_Private
 
 extension Result {
-    init(expected: MBXExpected<AnyObject, AnyObject>) {
+    
```

Comment:
Is this empty line needed?

Reply:
Not too critical, but I try to follow style which adds empty line for all properties, methods etc, which were declared right inside another class or struct. I think we have similar style in most of the other places.